src/main/java/xyz/nucleoid/parties/PartyManager.java

```diff
@@ -105,12 +106,11 @@ public PartyResult kickPlayer(PlayerRef owner, PlayerRef player) {
         return PartyResult.err(PartyError.NOT_IN_PARTY);
     }
 
-    public PartyResult acceptInvite(PlayerRef player, PlayerRef owner) {
+    public PartyResult acceptInvite(PlayerRef player, Party party) {
```

Comment:
Nit: `@Nullable` party?

Reply:
Fixed in 57295c33496e22f0a5564099a437217d2b9dfdca